competencies/roles/DeveloperLevelMapping.md

```diff
@@ -7,7 +7,7 @@ The three categories of competencies can be used to map to typical job titles sp
 | SW Dev I (junior)         |                                                              |                                |  New Grad, or first one or two years |
 | SW Dev II                 |                                                              |  75% emerging, 25% proficient  | Integrated into the software practice at Properly |
 | SW Dev III (intermediate) | Proficient: "Tech Design, Implementation and Code Ownership" | 50% emerging, 50% proficient   | Strong in knowledge and execution |
-| SW Dev IV (senior I)      | Proficient: "System Design"                                  | 50% proficient; some authority | May be expert or generalist  |
-| SW Dev V (senior II)      | Proficient: "Communication"                                  | 75% proficient; some authority | Expanded impact across teams / org |
-| Principal SW Dev          | Proficient: "Technical Change"                               | 50% Proficient; 50% Authority  | Strong self awareness, Strong on general areas such as business acumen |
+| SW Dev IV (senior I)      | Proficient: "System Design"                                  | 75%  proficient                | May be expert or generalist  |
+| SW Dev V (senior II)      | Proficient: "Communication"                                  | 25% authority                  | Expanded impact across teams / org |
```

Comment:
Is this role 75% proficient + 25% authority?
Or is it 75% proficient or greater?

Reply:
75% proficient or greater was my intention, with the assumption that normally that will mean as "junior" as all the rest being emerging. But in practice sometimes authority emerges while still emerging in other areas of a role.   

Do you think that makes sense and will make the progression natural?

Reply:
This would still be a 2 proficiencies improvement to go from IV -> V.  Arguably two proficients would turn into authority.

I think this is still fair but this makes the gab between IV -> V is smaller than I initially understand it.

Reply:
For clarity, I think this should be rephrased as `75% proficient or greater with 25% authority` or similar.

Reply:
If I'm understanding the suggestion, I'd phrase this as `50% proficient; 25% authority;` being the minimum requirement here (i.e. become an authority in 2 of your proficient areas).  The gap feels reasonable to me, given we're saying the authority boxes are very hard to get into

Reply:
Updated with "75% percent proficient or greater" 
![image](https://user-images.githubusercontent.com/36340630/101519242-f4133b00-3950-11eb-80f9-75cf1e9d63c9.png)

Note that is still intended to be a mix of emerging in that so feels like there still us a meaningful step to level 5; am I misunderstanding the scenario you are thinking about?

Reply:
Suggestions above are for the SW Dev V line being unclear - e.g. if 25% authority, what is a valid mix of the rest?